### PR TITLE
Create contact me page

### PR DESCRIPTION
**Pull request by Laylooo**
Added Contact me page to index.html;
In style.css file given style for this attribute.
Added link to Formspree service

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,6 +209,36 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+						</li>
+
+						<li class="textarea">
+							<label for="msg"></label>
+							<textarea id="msg" name="user_message" placeholder="Write your message here"
+								maxlength="500" required></textarea>
+						</li>
+
+					</ul>
+					<button class="button" type="submit">Get in touch</button>
```
Comment (Davidosky007) [inline index.html:237]: The Button is missing the necessary interactions eg(hover, enable etc.) please fix this as its part of the project requirement

File: index.html [hunk 2]
```diff
@@ -209,6 +209,36 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+
+				</form>
+			</div>
+		</section>
```
Comment (Davidosky007) [inline index.html:241]: I suggest you stick to the Figma design as much as possible. This is how your contact section looks like, please compare and make the necessary changes 
Screenshot (73)

Figma Design:
Screenshot (74)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Laylooo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
# I left comments below please make necessary changes 🙏 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I strongly suggest you fix all broken social icons 
Screenshot (72)

Screenshot (70)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Laylooo) [reply index.html:237]: I made it enabled in the style.css file should I have to do it again or am I doing a mistake. Thanks in advance 😊

**Review by Shaher-11 — APPROVED** [review]
Hello @Laylooo ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**